examples/translate_rectangle.ml

```diff
@@ -0,0 +1,18 @@
+open Graphics
+
+let translate_rectangle dx dy rectangle =
```

Comment:
You don't need this function, in the example you can do

```ocaml
let r1 = rectangle 200 100 in
let r2 = translate 100 0 r1 in
```

the `translate` function is polymorphic, so it can handle any shape being passed as an argument.

Reply:
Same applies for other examples ^^